waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoringConfigurationTest.java

```diff
@@ -0,0 +1,56 @@
+package com.hotels.bdp.waggledance.metrics;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.io.Closeable;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.codahale.metrics.MetricRegistry;
+
+import com.hotels.bdp.waggledance.conf.GraphiteConfiguration;
+
+@RunWith(MockitoJUnitRunner.class)
+public class MonitoringConfigurationTest {
+
+  private final MetricRegistry metricRegistry = new MetricRegistry();
+  private @Mock GraphiteConfiguration graphiteConfiguration;
+
+  private MonitoringConfiguration monitoringConfiguration;
+
+  @Before
+  public void setUp() {
+    monitoringConfiguration = new MonitoringConfiguration();
+    monitoringConfiguration.setMetricRegistry(metricRegistry);
+    monitoringConfiguration.setGraphiteConfiguration(graphiteConfiguration);
+  }
+
+  @Test
+  public void unenabledGraphiteConfigurationShouldGetOneReporter() {
```

Comment:
unenabledGraphiteConfigurationShouldGetOneReporter -> disabledGraphiteConfiguration

("disabled" better than "unenabled" and also if you don't say what the expected behaviour is in the method name its one less thing to modify if the behaviour changes)